.github/auto_assign.yml

```
@@ -0,0 +1,11 @@
+# auto-assign-action の設定ファイル
+# https://github.com/kentaro-m/auto-assign-action
+
+# PR作成者をアサインするか
+addAssignees: true
+
+# ドラフトPRでも実行するかどうか
+runOnDraft: true
+
+# "author" を指定すると、PR作成者をアサインする
+addAssignees: author
```

YAMLlint (1.37.1)

[error] duplication of key "addAssignees" in mapping

(key-duplicates)